### PR TITLE
TiCDC: Update ticdc-overview.md

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

Update ticdc-overview.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3330, https://github.com/pingcap/docs-cn/pull/3141, https://github.com/pingcap/docs-cn/pull/3352/, https://github.com/pingcap/docs-cn/pull/3356, https://github.com/pingcap/docs-cn/pull/2909
- Other reference link(s):

### CODE REVIEW

File: ticdc/ticdc-overview.md [hunk 1]
```diff
@@ -41,13 +45,29 @@ Currently, the TiCDC sink component supports replicating data to the following d
...
 To replicate data to TiDB or MySQL, you must ensure that the following requirements are satisfied to guarantee data correctness:
 
 - The table to be replicated has the primary key or a unique index.
 - If the table to be replicated only has unique indexes, each column of at least one unique index is explicitly defined in the table schema as `NOT NULL`.
+
+### Unsupported scenarios
+
+Currently, The following scenarios are not supported:
+
+- The replication of partition tables.
```
Comment (amyangfei) [inline ticdc/ticdc-overview.md:59]:
```suggestion
```
This feature is supported now, we should also update docs-cn

File: ticdc/ticdc-overview.md [hunk 2]
```diff
@@ -41,13 +45,29 @@ Currently, the TiCDC sink component supports replicating data to the following d
...
+- The TiKV cluster that uses RawKV alone.
+- The [new framework for collations](/character-set-and-collation.md#new-framework-for-collations) in TiDB v4.0. Before you enable this feature, make sure that the downstream cluster is the TiDB cluster with the same collation as those in the upstream; otherwise, inconsistent collations lead to the issue that the data cannot be located.
+- [The DDL operation `CREATE SEQUENCE`](/sql-statements/sql-statement-create-sequence.md) and the [SEQUENCE function](/sql-statements/sql-statement-create-sequence.md#sequence-function) in TiDB v4.0. When the upstream TiDB uses SEQUENCE, TiCDC ignores SEQUENCE DDL operations/functions performed upstream. However, DML operations using SEQUENCE functions can be correctly replicated.
```
Comment (ran-huang) [inline ticdc/ticdc-overview.md:62]:
```suggestion
- The [DDL operation `CREATE SEQUENCE`](/sql-statements/sql-statement-create-sequence.md) and the [SEQUENCE function](/sql-statements/sql-statement-create-sequence.md#sequence-function) in TiDB v4.0. When the upstream TiDB uses `SEQUENCE`, TiCDC ignores `SEQUENCE` DDL operations/functions performed upstream. However, DML operations using `SEQUENCE` functions can be correctly replicated.
```

File: ticdc/ticdc-overview.md [hunk 3]
```diff
@@ -41,13 +45,29 @@ Currently, the TiCDC sink component supports replicating data to the following d
...
+- The [TiKV Hibernate Region](https://github.com/tikv/tikv/blob/master/docs/reference/configuration/raftstore-config.md#hibernate-region). TiCDC prevents the Region from entering the hibernated state.
+- The scheduling of existing replication tables to new TiCDC nodes, after the capacity of the TiCDC cluster is scaled out.
+- The replication of Black and white table lists.
+
+## Manage TiCDC Cluster and Replication Tasks
+
+For details, see [Manage TiCDC Cluster and Replication Tasks](/ticdc/manage-ticdc.md).
+
```
Comment (ran-huang) [inline ticdc/ticdc-overview.md:70]: I can't find the troubleshooting file in the repo. Is it a P2 doc? 
If so, ignore this comment. We can add this link after troubleshoot-ticdc.md is ready.
```suggestion

## TiCDC troubleshooting

For the common questions and corresponding solutions during the use of TiCDC, refer to [TiCDC Troubleshooting](/ticdc/troubleshoot-ticdc.md)

```

File: ticdc/ticdc-overview.md (hunk 1 above)
Comment (toutdesuite) [reply ticdc/ticdc-overview.md]: updated in https://github.com/pingcap/docs-cn/pull/3552

**Pull request merged**